package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "distributed-wikipedia-mirror",
+  "description": "Mirror wikipedia on IPFS",
+  "version": "0.1.0",
```

We don't want accidental NPM publish to happen:
```suggestion
  "version": "0.1.0",
  "private": true,  
```